referrer-policy/no-referrer-policy/no-referrer-policy.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Check that subresource gets the referrer URL when no explicit
```

Comment:
I'd suggest not adding another frame, but splitting it out into a separate test. Multiple frames introduces flakiness (Maybe frame 1 loads first? Maybe frame 2 hangs for an hour?), single-frame tests are clearer and test only a single thing. That's a good goal to aim for.